CrazyCanvas/Include/GUI/HUDGUI.h

```diff
@@ -63,8 +64,8 @@ class HUDGUI : public Noesis::Grid
 private:
 	GameGUIState m_GUIState;
 
-	Noesis::Border* m_pWaterAmmoRect;		
-	Noesis::Border* m_pPaintAmmoRect;		
+	Noesis::Image* m_pWaterAmmoRect = nullptr;		
+	Noesis::Image* m_pPaintAmmoRect;
```

Comment:
Why does `m_pWaterAmmoRect ` need to be nullptr but `m_pPaintAmmoRect` does not? If there is no strong motivation I suggest to set either all to Noesis resources to nullptr either here or in `Init`

Reply:
Done